app/managers/global_event_handler.ts

```diff
@@ -18,9 +19,11 @@ import NetworkManager from '@managers/network_manager';
 import WebsocketManager from '@managers/websocket_manager';
 import {getCurrentUser} from '@queries/servers/user';
 import EphemeralStore from '@store/ephemeral_store';
-import {LaunchType} from '@typings/launch';
 import {deleteFileCache} from '@utils/file';
 
+import type {jsAndNativeErrorHandler} from '@typings/global/error_handling';
+import type {LaunchType as LaunchTypeType} from '@typings/launch';
```

Comment:
Because we have the constant, which is the enum, and the type, which is the derived from the constant. In this file we have to define a variable as the type, and assign values from the constant (line 122).

If we try to use the constant as a type, we get the "x refers to a value but is being used as a type here" error. In order to properly set it we would have to do the same we have done in the typings file, so... better import the file from typings.

Reply:
ok understood, then can we change this to be `LaunchType` and the constant as `LaunchTypeConstant`? I think it reads better, what do you think?

Reply:
Actually now it seems that the constant is called Launch, so this can be LaunchType instead of LaunchTypeType